tap_zendesk/streams.py

```diff
@@ -31,6 +32,16 @@
     'checkbox': 'boolean',
 }
 
+def update_authorization_headers(config, headers):
+    if "access_token" in config:
+        headers['Authorization'] = 'Bearer {}'.format(config["access_token"])
+    else:
+        basic_auth_string = '{}/token:{}'.format(config["email"], config["api_token"])
+        encoded_basic_auth = base64.b64encode(basic_auth_string.encode("utf-8")).decode("utf-8")
+        headers['Authorization'] = 'Basic {}'.format(encoded_basic_auth)
+    return headers
+
+
```

Comment:
this method is a part of the APi client, so this func should belong to the http.py file.

Reply:
okay.